.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install dependencies
+      run: npm ci
+    - name: Check formatting
+      run: npm run format-check
+    - name: Build
+      run: npm run build --if-present
+    - name: Run Tests
+      run: npm test
```

Comment:
Maybe use the other call here, the one which isn't silent. (see description for this PR for what I mean)